package.json

```diff
@@ -6,22 +6,29 @@
    "scripts": {
       "dev": "vite",
       "build": "tsc && vite build",
-      "preview": "vite preview"
+      "preview": "vite preview",
+      "test": "NODE_NO_WARNINGS=1 mocha"
```

Comment:
what is that `NODE_NO_WARNINGS=1` for?

Reply:
It silences the following log:

```
(node:9196) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("ts-node/esm", pathToFileURL("./"));'
(Use `node --trace-warnings ...` to show where the warning was created)
```

Reply:
So lets find a solution that is not this experimental node loader so this error won't be thrown, and then delete this.

Reply:
It silences the following log:

```
(node:9196) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("ts-node/esm", pathToFileURL("./"));'
(Use `node --trace-warnings ...` to show where the warning was created)
```

Reply:
found another way without experimental node loader?

Reply:
I set with Sagiv for long enough we don't think theres a better option. I can go to Avi with that if its important.